tidb_cloud/README.md

```diff
@@ -0,0 +1,43 @@
+# TiDB Cloud Integration
+
+## Overview
+
+[TiDB Cloud][1] makes deploying, managing and maintaining your TiDB clusters even simpler with a fully managed cloud instance that you control through an intuitive console. 
+
+TiDB Cloud allows developers and DBAs with little or no training to handle once-complex tasks such as infrastructure management and cluster deployment with ease, to focus on your applications, not the complexities of your database.
+
+The Datadog integration is an observability feature of TiDB Cloud that exports metrics of TiDB Cloud clusters to your Datadog platform.
```

Comment:
```suggestion
[TiDB Cloud][1] is a fully managed cloud service of TiDB, an open-source database.

Use the TiDB Cloud Datadog integration to export metrics from TiDB Cloud clusters to Datadog.
```

Reply:
Sorry for removing your writing, but we don't allow marking-type prose in the integrations docs.